web-ui/src/components/team-member/TeamMemberSelect.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import Avatar from "@material-ui/core/Avatar";
+
+import "./TeamMemberSelect.css";
+
+const TeamMemberSelect = (props) => {
+  const { teamMembers, onChange, singleSelect = false } = props;
+  const [filteredTeamMembers, setFilteredTeamMembers] = useState(teamMembers);
```

Comment:
How will this change due to our context work? Not saying at all that you need or should do that now.

Reply:
Whichever component calls TeamMemberSelect will have to do the team api call and pass the members down